src/app/components/events/bet-type-form/bet-type-form.component.html

```diff
@@ -0,0 +1,120 @@
+<form [formGroup]="form">
+  <div formArrayName="betTypes">
+    <div
+      *ngFor="let betType of betTypes.controls; let i = index"
+      [formGroupName]="i"
+      class="bet-container"
+    >
+      <div class="bet-title">
+        {{
+          betType.get("name")?.value ? betType.get("name")?.value : "New bet"
+        }}
+      </div>
+      <div class="bet-name-container">
+        <mat-form-field color="accent" appearance="outline">
+          <mat-label>Bet name</mat-label>
+          <input
+            maxlength="50"
+            matInput
+            placeholder="Red cards"
+            value="2"
+            formControlName="name"
+          />
+          <mat-error *ngIf="form.errors?.['insufficientOptions']">
+            Must add at least 2 options.
+          </mat-error>
+        </mat-form-field>
+        <mat-form-field appearance="outline" color="accent">
+          <mat-label>Bet type</mat-label>
+          <mat-select panelClass="select-dropdown" formControlName="type">
+            <mat-option value="NUMERIC">Numeric</mat-option>
+            <mat-option value="MULTIPLE_CHOICE">Multiple choice</mat-option>
+            <mat-option value="BOOLEAN">Boolean</mat-option>
+            <mat-option value="STRING">Text</mat-option>
+          </mat-select>
+        </mat-form-field>
+        <mat-icon fontIcon="delete" (click)="removeBetType(i)"></mat-icon>
+      </div>
+
+      <div
```

Comment:
double check for betType.get('type')?.value === 'MULTIPLE_CHOICE'" here and on line 40. You can make that check only once in a parent div

Reply:
This entire file is almost completely changed in newer versions, because bets don't even have types (such as MULTIPLE_CHOICE) anymore. Should I still try and remove the double check in this version ?